### PR TITLE
Friendlier landing page

### LINKED ISSUE #41: Improve landing page
The [landing page](https://docs.pickit3d.com/docs/pickit/en/latest/) of our knowledge base is **very dry**, pure text. Proposals to improve this:
- Prominently display an inspiring picture of what Pickit can do.
- Add a warm welcome message along the lines of "Welcome to the Pickit knowledge base! Here you can find this and that. The left panel guides you through different sections blah blah. If you can't find it here, please contact us at support@pickit3d.com. Happy reading!"

### PR DESCRIPTION
**Pull request by adolfo-rt**
Fixes #41.

This PR does the following:
- Reduce the landing page to a one-pager.
  - Don't repeat left panel TOC by using `:hidden:`.
  - Link to `pickit3d.com`. Don't repeat content that is better explained there.
- Point to support email.
- Give a warm and friendly welcome to the reader.
- Show @bartpickit :smiley:

This is how it looks, so you don't have to render it locally to give feedback. The left panel remains unchanged. Bullet points below correspond to the green headers in the left panel.

image

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
+=====================================
 
-Pickit is a 3D vision application for picking a broad family of products in any
-location with any robot. Pickit can be used for lcoating objects during
-palletizing, depalletizing, order picking, bin-picking, machine tending or
-assembly tasks with a robot or machine.
+Here you can find everything you need to know about
+`Pickit <https://www.pickit3d.com/>`__, a plug and play 3D camera for your robot
+that allows it to pick parts of different shapes, materials and sizes.
+The panel to the left of the page guides you though the different topics of your
```
Comment (anaritaflp) [inline docs/index.rst]:
```suggestion
The panel to the left of the page guides you through the different topics of your
```

File: docs/index.rst [hunk 2]
```diff
@@ -1,20 +1,38 @@
-Pickit Documentation
-====================
+Welcome to the Pickit knowledge base!
+=====================================
 
-Pickit is a 3D vision application for picking a broad family of products in any
-location with any robot. Pickit can be used for lcoating objects during
-palletizing, depalletizing, order picking, bin-picking, machine tending or
-assembly tasks with a robot or machine.
+Here you can find everything you need to know about
```
Comment (anaritaflp) [inline docs/index.rst]: I would make the very first sentence very broad and commercial, and add a second sentence with details on the advantages of Pickit. For example:

```
Here you can find everything you need to know about Pickit, a plug-and-play 3D vision 
solution that gives eyes to your robot. Detecting a large variety of objects of different 
shapes, materials and sizes, Pickit allows automating any type of picking application, 
such as bin picking, depalletizing, machine loading, pick and place, and kitting and 
assembly.
```

File: docs/index.rst (hunk 1 above)
Comment (anaritaflp) [inline docs/index.rst]: Add a blank line before this sentence. Then the first paragraph is a short description of Pickit and the second paragraph is about how to navigate in the KB and how it's structured.

File: docs/index.rst (hunk 2 above)
Comment (anaritaflp) [reply docs/index.rst]: I'm not sure if we still use the `giving eyes to robots` expression though...

Comment (adolfo-rt) [reply docs/index.rst]: The current first phrase is also the first one from our website. I'm ok with re-adding something about applications, but minimal. That is the scope of the website.

**Comment by achim-k** [comment]
Closing as it has been moved to a different repo.

**Pull request closed without merging**